mobile/src/screens/auth/components/label/styles.ts

```diff
@@ -8,7 +8,7 @@ const styles = StyleSheet.create({
     fontSize: 24,
     fontWeight: 'bold',
     letterSpacing: 3,
-    backgroundColor: AppColor.BLUE_200,
+    backgroundColor: AppColor.BLUE_300,
```

Comment:
There're lots of similar small changes in styles. Are these purposeful changes or just merger conflicts?

Reply:
Merged conflict yes)